configure/CMakeLists.txt

```diff
@@ -174,34 +192,14 @@ if(${FAILED})
     message(SEND_ERROR " No omniidl was found! rv=${FAILED}")
 endif()
 
-#assuming build platform == target
-set(PLATFORM 32)
-if(WIN32)
-    if(CMAKE_CL_64)
-        set(PLATFORM 64)
-    endif(CMAKE_CL_64)
-else(WIN32)
-    if(${CMAKE_SIZEOF_VOID_P} EQUAL 8)
-        set(PLATFORM 64)
-    endif()
-endif(WIN32)
-
 if(NOT WIN32)
     include(CheckCXXCompilerFlag)
     # C++17 and higher support is currently not possible as omniorb uses
     # throw specifications and these are not supported anymore.
     CHECK_CXX_COMPILER_FLAG("-std=c++14" COMPILER_SUPPORTS_CXX14)
-    CHECK_CXX_COMPILER_FLAG("-std=c++11" COMPILER_SUPPORTS_CXX11)
-    CHECK_CXX_COMPILER_FLAG("-std=c++0x" COMPILER_SUPPORTS_CXX0X)
     CHECK_CXX_COMPILER_FLAG("-Og" COMPILER_SUPPORTS_OG)
     if(COMPILER_SUPPORTS_CXX14)
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
-    elseif(COMPILER_SUPPORTS_CXX11)
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
-    elseif(COMPILER_SUPPORTS_CXX0X)
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x")
-    else()
-        message("The compiler ${CMAKE_CXX_COMPILER} has no C++11/14 support. You may not benefit from performance optimizations.")
     endif()
```

Comment:
What `-std` flag will be used for compilers that do not support c++14, e.g. gcc 4.8.1)? If I recall, `-std=c++11` was never a default setting in gcc and if this is true, the compiler will run in c++98 mode, right? On Debian 8 we compile in c++14 mode so actually there is no job in CI that checks the c++11 mode.

Reply:
I think @bourtemb answered this question. Resolving.